scripts/manifest.ts

```diff
@@ -75,12 +75,12 @@ const baseManifest: Manifest.WebExtensionManifest = {
 function chromeManifestV3(): Manifest.WebExtensionManifest {
 	return Object.assign({}, baseManifest, {
 		"manifest_version": 3,
-		"minimum_chrome_version": "88",
+		"minimum_chrome_version": "102",
 		"oauth2": {
 			// The Chrome identity client ID for Chrome ONLY
 			"client_id": "876467817034-al13p9m2bphgregs0rij76n1tumakcqr.apps.googleusercontent.com",
 			"scopes": [
-				"https://www.googleapis.com/auth/drive.readonly"
+				"https://www.googleapis.com/auth/drive.file"
```

Comment:
cc #354 

I caused a regression here by reverting an earlier change to `drive.readonly` (the wrong permission).